CONTRIBUTING.md

```diff
@@ -1,1199 +1,3 @@
-
-### Repositories
-
-#### Naming
-
-* Project name must not contain odoo or openerp
-* Project name for localization is "l10n-belgium" for Belgium
-* Project name for connectors is "connector-magento" for Magento connector
-
-#### Branch configuration
-Python packages to install, must be preferably, define in requirements.txt than travis.yml file.
-Requirements.txt avoid to repeat packages in all travis.yml files of repositories in case of using with oca_dependencies.txt file.
-
-### Issues
-
-* Issues are used for blueprints and bugs.
-
-## Differences With Odoo Guidelines
-
-Not the entire Odoo guidelines fit OCA modules needs. In many cases rules need
-to be more stringent. In other cases, conventions are improved for better
-maintainability in an ecosystem of many smaller modules.
-
-The differences include:
-
-* [Module Structure](#modules)
-    * Using one file per model
-    * Separating data and demo data xml folders
-    * Not changing xml_ids while inheriting
-    * Add guideline to use external dependencies
-    * Define a separated file for installation hooks
-* [XML](#xml-files)
-    * Avoid use current module in xml_id
-    * Use explicit `user_id` field for records of model `ir.filters`
-* [Python](#python)
-    * Use Python standards
-    * Fuller PEP8 compliance
-    * Use ``# coding: utf-8`` or ``# -*- coding: utf-8 -*-`` in first line
-    * Using relative import for local files
-    * More python idioms
-    * A way to deal with long comma-separated lines
-    * Hints on documentation
-    * Don't use CamelCase for model variables
-    * Use underscore uppercase notation for global variables or constants
-* [SQL](#sql)
-    * Add section for No SQL Injection
-    * Add section for don't bypass the ORM
-    * Add section for never commit the transaction
-* [Field](#field)
-    * A hint for function defaults
-    * Use default label string if is possible
-    * Add the inverse method pattern
-* [Tests Section Added](#tests)
-* [Git](#git)
-    * No prefixing of commits
-    * Default git commit message standards
-    * Squashing changes in pull requests when necessary
-    * Use of present imperative
-* [Github Section](#github)
-* [Review Section](#review)
-
-# Backporting Odoo Modules
-
-Suggesting a backport of a module among an OCA repository is possible, but you
-must respect a few rules:
-
- * You need to keep the license of the module coded by Odoo SA
- * You need to add the OCA as author (and Odoo SA of course)
- * You need to make the module "OCA compatible" : PEP8, OCA convention and so
-   on so it won't break our CI like runbot, Travis and so.
- * You need to add a disclaimer in the Readme with the following text:
-```
-**This module is a backport from Odoo SA and as such, it is not included in the OCA CLA. That means we do not have a copy of the copyright on it like all other OCA modules.**
-```
+This document has moved to https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst 
```

Comment:
I have a question here: shouldnot we use the canonical URL instead?
If later on we generate with sphinx and static pages it might make sense.
Or we could provide both URL such as:
```
This document has moved to https:odoo-community.org/contributing [not sure about the exact location]
If you wish to contribute, please open a Pull Request to the source document available here: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst
```

Reply:
@elicoidal we don't have such a canonical url for this document so far. I could create one but so far it's manual. So I'd prefer we clarify the strategy about our website first. Currently the website links to this github document directly.

Reply:
Can't we use a redirect page on the OCA website, or even an URL shortener service? That would gice us flexibility.

Reply:
The current redirect mechanism on the OCA website is me maintaining redirections in the nginx config. Until we have something better, I propose we move forward with this PR. The only goal here is to avoid the same document being maintained in 2 different places.

Reply:
@sbidoul AFAIK, you can do the redirections through Odoo itself.

Reply:
@pedrobaeza I did not find it, do you have a pointer for me?

Reply:
@pedrobaeza that's a menu entry, so far I knew :)

What I think @dreispt wants is a short, stable url, so we need a kind of url redirector. The was website_seo_redirection but it's not migrated yet.